src/sentry/static/sentry/app/views/settings/organizationIntegrations/configureIntegration.tsx

```diff
@@ -137,12 +142,24 @@ class ConfigureIntegration extends AsyncView<Props, State> {
           </Form>
         )}
 
-        {integration.dynamicDisplayInformation?.configure_integration?.instructions.map(
-          (instruction, i) => (
-            <Alert type="info" key={i}>
-              <span dangerouslySetInnerHTML={{__html: singleLineRenderer(instruction)}} />
-            </Alert>
-          )
+        {instructions?.length === 1 ? (
+          <Alert type="info">
```

Comment:
@EvanPurkhiser Nit: you can put the `Alert` tag outside the ternary

Reply:
Good call